.github/CODEOWNERS

```diff
@@ -39,6 +39,7 @@ web/gui/ @jacekkolasa @cosmix
 # Ownership by filetype (overwrites ownership by directory)
 *.md @cosmix @joelhans
 *.am @cosmix @Ferroin @knatsakis @ncmans @prologic
+Dockerfile.* @Ferroin @knatsakis @ncmans @prologic
```

Comment:
Please add me as code owner wherever you made changes.

Reply:
Done